newIDE/app/src/Utils/GDevelopServices/Asset.js

```diff
@@ -278,3 +293,30 @@ export const isPixelArt = (
     return tag.toLowerCase() === 'pixel art';
   });
 };
+
+export const isPrivateAsset = (
+  assetOrAssetShortHeader: AssetShortHeader | Asset
+): boolean => {
+  return assetOrAssetShortHeader.previewImageUrls[0].startsWith(
+    'https://private-assets'
```

Comment:
nit: maybe better be consistent with isPrivateAssetUrl and against the two base urls?